lib/jenkins_pipeline/pipeline_builder.rb

```diff
@@ -1,12 +1,12 @@
 module JenkinsPipeline
   class PipelineBuilder
-    def build(jobs_in_folder, pipeline)
+    def build(pipeline, jenkins_jobs)
       jobs = []
       pipeline_instance = Pipeline.new pipeline["name"]
 
       pipeline["jobs"].each_with_index do |job, index|
-        job_in_folder = jobs_in_folder["jobs"].select {|job_in_folder| job["ci_name"] == job_in_folder["name"] }.first
-        job_instance = Job.new(job_in_folder, job["name"])
+        jenkins_job = jenkins_jobs["jobs"].select {|jenksin_job| job["ci_name"] == jenksin_job["name"] }.first
```

Comment:
Line is too long. [110/80]<br>Use `find` instead of `select.first`.<br>Space between { and | missing.